docs/rgb_decomposition.md

```diff
@@ -0,0 +1,49 @@
+
+### Interpreting the Images
+
+Calm water generally has very low returns in all polarizations, and areas with very low backscatter will appear blue. Desert landscapes have a backscatter signature similar to that of water, however, so care should be taken in interpretation of the blue color based on the location. 
+
+The green band is determined by the magnitude of the cross-pol returns, which indicates the extent of volume scattering. One of the most common volume scatterers is vegetation, and areas with high cross-pol returns appear more green in the decomposition. There are other volume scatterers, such as glacial ice, that may also appear as green, so interpretation will be site-dependent.
+
+The red channel is used to indicate areas that are neither water nor vegetation. Urban areas and other locations with relatively low levels of volume scattering will appear more red in this decomposition.
+
+### Decomposition Calculations
+
+Note that dual-pol image acquisition may provide either VV/VH returns or HH/HV returns, depending on the sensor mode. The same decomposition approach can be applied to either mode; use the co-pol (S<sub>CP</sub>) and cross-pol (S<sub>XP</sub>) pixel values available for the specific dataset. 
+
+To calculate the RGB values on a pixel-by-pixel basis, the data is first separated into surface (P<sub>r</sub>) and volume (P<sub>v</sub>) scattering components as follows:
+
+P<sub>r</sub> = S<sub>CP</sub> - 3 S<sub>XP</sub>
+
+P<sub>v</sub> = 4 S<sub>XP</sub>
+
+A spatial mask is generated using a threshold (*k*) applied to the P<sub>v</sub> data so that pixels with values below the threshold are included in the blue channel mask (M<sub>B</sub>) and those above the threshold are included in the red band mask (M<sub>R</sub>). Threshold values typically range between -22 and -25 dB, and the default for HyP3 processing is -24 dB.
+
+M<sub>B</sub> = P<sub>v</sub> < *k*
+
+M<sub>R</sub> = P<sub>v</sub> >= *k*
+
+These spatial masks are then applied to the P<sub>r</sub> data to assign the pixels to either the red or the blue band.
+
+P<sub>B</sub> = P<sub>r</sub> M<sub>B</sub>
+
+P<sub>R</sub> = P<sub>r</sub> M<sub>R</sub>
+
+Finally, the intensity values for each band are multiplied by specific scalars to appropriately stretch the dynamic range of each band to create an RGB:
+
+z = arctan ( 2 (S<sub>CP</sub> -  S<sub>XP</sub>) <sup>0.5</sup> ) 2 / 𝜋
+
+a<sub>R</sub> = 254 ( 2 (P<sub>R</sub>) <sup>0.5</sup> + z ) + 1 
+
+a<sub>G</sub> = 254 ( 3 / 2 (P<sub>v</sub>) <sup>0.5</sup> + 2z ) + 1
+
+a<sub>B</sub> = 254 ( 2 (P<sub>B</sub>) <sup>0.5</sup> + 5z) + 1 
```

Comment:
Adjusted equation for z to assume power instead of amplitude for input RTC scale.
Rearranged equations for readability.

Reply:
I think you have an extra 2 in the Z, should be:

```
z = arctan ( (S<sub>CP</sub> -  S<sub>XP</sub>) <sup>0.5</sup> ) 2 / 𝜋
```

but the rest look fine. I prefer a 1 + at the front, but it doesn't matter if it's + 1 and the end

Reply:
I went with the equation for z that you posted in the comments, just removed the straight brackets and square roots:
z = arctan( 2 (|S<sub>cp</sub>| <sup>2</sup> -  |S<sub>xp</sub>| <sup>2</sup>) <sup>0.5</sup> ) * 2 / 𝜋
Please go ahead and remove the 2 if it's not supposed to be there.

Reply:
Woops, that was my typo then:
https://github.com/ASFHyP3/hyp3-lib/blob/2fcbf79556b10ea53c75ac5b98188391d0c93935/hyp3lib/rtc2color.py#L89

I'll push a fix